src/main/java/mate/academy/bookstorev2/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.bookstorev2.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Look up possible side effects of using lombok's data on JPA entities, fix to just @ Getter, @ Setter , @ NoArgsConstructor for now